src/utils/components/DiskModal/DiskFormFields/DetachHotplugDiskCheckbox.tsx

```diff
@@ -22,7 +22,7 @@ const DetachHotplugDiskCheckbox: React.FC<DetachHotplugDiskCheckboxProps> = ({
     <FormGroup
       fieldId="detach-hotplug-disk"
       helperText={t(
-        'Leave this option unchecked to make your hotplugged disk a permanent part of your virtual machine configuration.',
+        'Leave this option unchecked to make your hotplugged disk a permanent part of your VirtualMachine configuration.',
```

Comment:
please rebase your PR and see this entire file is being removed by this PR:
https://github.com/kubevirt-ui/kubevirt-plugin/pull/386